h/accounts/views.py

```diff
@@ -38,6 +38,70 @@ def ajax_payload(request, data):
     return payload
 
 
+def to_xhr_response(request, non_xhr_result, form):
```

Comment:
Ok. I'm not really sure why they couldn't have moved in this PR, as it seems like a fairly straightforward change... but I'm not going to kick up a fuss about it.